_includes/social_links/dribbble.html

```diff
@@ -0,0 +1 @@
+<li><a href="https://dribbble.com/{{ site.dribbble_username }}"><svg class="svg-icon" xmlns="http://www.w3.org/2000/svg"><use xlink:href="/assets/icons.svg#dribbble"></use></svg> <span class="username">{{ site.dribbble_username }}</span></a></li>
```

Comment:
why add the `<li>` tag in the include file? What if you want to use the SVG outside of a list ?